src/hotspot/cpu/riscv/macroAssembler_riscv.hpp

```diff
@@ -1321,13 +1321,13 @@ class MacroAssembler: public Assembler {
   // to use a 2nd scratch register to hold the constant. so, an address
   // increment/decrement may trash both t0 and t1.
 
-  void increment(const Address dst, int64_t value = 1, Register tmp1 = t0, Register tmp2 = t1);
+  void increment (const Address dst, int64_t value = 1, Register tmp1 = t0, Register tmp2 = t1);
```

Comment:
To align with the pattern in this file, extra space should be removed.

Reply:
Fixed. Thanks.